tests/fixtures/basic-js/.sauce/config.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1alpha
+kind: folio
```

Comment:
@dpgraham should `kind` be: `playwright-folio`?

Reply:
Do we need to mention `folio` prominently at all? I think we can get away with just `playwright` for quite a long while.